### PR TITLE
Made fixes to the welcome email template for issue #1228

### PR DESCRIPTION
**Pull request by Coop56**
This pull request should fix the issues we had on the welcome email template.
- Fixed the image link.
- Fixed the get skills link.
- Fixed the get involved link.
- Added the youtube embed

### CODE REVIEW

File: mapstory/templates/account/email/welcome_message.html [hunk 1]
```diff
@@ -2,80 +2,126 @@
...
+
+                                <p style="margin:8px 0">There's three ways to share on MapStory. Just remember <b>A, B, C</b>. First, you can
+                                    <b>add</b> new spatially and temporally enabled datasets, which we call StoryLayers. Second, you can <b>build</b>
+                                    MapStories using StoryLayers and other narrative elements that you would like. Finally, you can <b>curate</b>
+                                    content that appears on MapStory to improve its quality and accuracy.</p>
+
+                                <p style="margin:8px 0">Start <a style="color:#f46817" href="http://mapstory.org/">sharing on MapStory</a>.</p>
+
+                                <h3 style="color:#f46817; font-size:18px; font-weight:normal; margin:32px 0 8px 0;"><a
+                                        href="http://www.mapstory.org/{% url 'getpage' 'involved' %}">Build the Community</a></h3>
```
Comment (davisc) [inline mapstory/templates/account/email/welcome_message.html]: See pull request for the best way to handle site urls https://github.com/MapStory/mapstory-geonode/pull/194 {{ current_site.url }}

**Comment by Coop56** [comment]
Ok I changed the urls over to using the current site domain tag instead and pushed back up @davisc.

File: mapstory/templates/account/email/welcome_message.html [hunk 2]
```diff
@@ -2,80 +2,126 @@
...
+
+                                <p style="margin:8px 0">There's three ways to share on MapStory. Just remember <b>A, B, C</b>. First, you can
+                                    <b>add</b> new spatially and temporally enabled datasets, which we call StoryLayers. Second, you can <b>build</b>
+                                    MapStories using StoryLayers and other narrative elements that you would like. Finally, you can <b>curate</b>
+                                    content that appears on MapStory to improve its quality and accuracy.</p>
+
+                                <p style="margin:8px 0">Start <a style="color:#f46817" href="{{ current_site.url }}">sharing on MapStory</a>.</p>
+
+                                <h3 style="color:#f46817; font-size:18px; font-weight:normal; margin:32px 0 8px 0;"><a
+                                        href="{{ current_site.url }}{% url 'getpage' 'involved' %}">Build the Community</a></h3>
```
Comment (davisc) [inline mapstory/templates/account/email/welcome_message.html]: replace with current_site.domain...

**Comment by Coop56** [comment]
My bad @davisc - they are all .domain now.

**Pull request merged**